hydra-chain-observer/src/Hydra/ChainObserver.hs

```diff
@@ -165,7 +165,7 @@ chainSyncClient tracer networkId startingPoint observerHandler =
       }
 
   clientStIdle :: UTxO -> ClientStIdle BlockType ChainPoint ChainTip m ()
-  clientStIdle utxo = SendMsgRequestNext (clientStNext utxo) (pure $ clientStNext utxo)
+  clientStIdle utxo = SendMsgRequestNext (pure ()) (clientStNext utxo)
```

Comment:
Interesting change they made here. This was certainly odd before where you would provide twice the next state.